test/verify/check-connection

```diff
@@ -537,6 +537,57 @@ class TestConnection(MachineCase):
 
         self.allow_journal_messages(".*Peer failed to perform TLS handshake")
 
+    @skipImage("Can't remove/upgrade packages on OSTree", "fedora-coreos")
+    def testWsPackage(self):
+        m = self.machine
+
+        if m.image.startswith("debian") or m.image.startswith("ubuntu"):
+            # clean up debug symbols, they get in the way of upgrading
+            m.execute("dpkg --purge cockpit-ws-dbgsym")
+        elif m.image.startswith("rhel"):
+            # subscription-manager-cockpit depends on cockpit-ws
+            m.execute("rpm --erase subscription-manager-cockpit")
+
+        def install():
+            if m.image.startswith("debian") or m.image.startswith("ubuntu"):
+                m.execute("dpkg --install /var/tmp/build-results/cockpit-ws_*.deb")
+            else:
+                # FIXME: current package complains about mode change on /run/cockpit and selinux
```

Comment:
What is it complaining about? Is this related to these changes?

Reply:
No, unrelated:
```
# rpm -V cockpit-ws
.M.......  g /run/cockpit
.M.......  g /var/lib/selinux/targeted/active/modules/200/cockpit
```
The first one has been there forever (/run/cockpit is just a %ghost), and I figure the other one was introduced with adopting our SELinux policy. We should fix these, of course, but for now this makes sure that  we don't introduce other complains than 'M'ode changes.

Reply:
@lnussel : You added this `%ghost %dir /run/cockpit` in  PR #13826. What was the purpose of that? It  breaks `rpm -V` validation, and this should really not be a directory that rpm should be concerned about. It's managed by systemd and cockpit's units entirely, not by rpm. Would it be ok to drop that again?

Reply:
Well you could remove it, it's not mandatory. But shouldn't hurt either. For now it mostly serves the purpose you see the, ie rpm -V. Why is there a disagreement about the mode of the directory?

Reply:
I have no idea.. On the actual system it's just standard root:root 0755. But as the rpm does not actually ship that dir, and `%ghost %dir` does not declare any explicit mode, I'm not sure what the expected mode even is.. But maybe it's not *really* about the mode, but it's complaining about the "g"?

     g %ghost file (i.e. the file contents are not included in the package payload).

then it seems pretty stupid to complain about this, because not shipping a file in the package is *exactly* what %ghost is for..

Reply:
The rpm does include a mode for the ghost file as it's actually created in %install. So from what I can tell it looks like doing the right thing here.

```
# rpm -qvl cockpit-ws|grep /run/cockpit$
drwxr-xr-x    2 root     root                        0 Apr 22 06:57 /run/cockpit
# rpm -Vv cockpit-ws|grep /run/cockpit$
.........  g /run/cockpit
# chmod 700 /run/cockpit
# rpm -V cockpit-ws
.M.......  g /run/cockpit
# rpm --restore cockpit-ws
# l -d /run/cockpit
drwxr-xr-x 2 root root 100  7. Mai 05:33 /run/cockpit/
# rpm -V cockpit
#
```

Reply:
I just checked the .spec again, and indeed that bit happens *only*  on SUSE:
```sh
# need this in SUSE as post build checks dislike stale symlinks
install -m 644 -D /dev/null %{buildroot}/run/cockpit/motd
```
So I suppose I'll conditionalize the `%ghost` as well, as the Fedora/RHEL packages don't ship bits in /run (and they really *really* shouldn't! Packaging /run makes me cringe. Are you sure you want that in OpenSUSE?)

Reply:
I believe this OpenSUSE-specific `install` hack should be obsolete since commit 5705709d416954c169e9af442ea3460ad48b6e9c . I'll send a PR after landing my `rpm -V` fix, and ask @lnussel for confirmation.

Reply:
%ghost in /run doesn't really package anything :-) Anyway feel free to remove, I have to maintain some openSUSE specific patches anyway (license, permission macro etc).
Installing links to /run in /etc makes me cringe btw :-) agetty, login, pam etc do support reading motd and issue from /run. Looks like just openssh doesn't.

Reply:
Right, but it keeps information about the expected modes/properties of the ghosted file/dir. Full ack about the installing links to /etc.. We still have some distros to support which don't yet support reading from /run/motd.d/ directly (that's fixed in most recent PAM, though). One of these days this will go away.